ecosystem/sep-0006.md

```diff
@@ -6,55 +6,95 @@ Title: Anchor/Client interoperability
+
+The deposit endpoint allows a wallet to get deposit information from an anchor, so a user has all the information needed to initiate a deposit. It also lets the anchor specify additional information (if desired) that the user must submit via the `/customer` endpoint to be able to deposit.
+
+### Request
+
+```
+GET TRANSFER_SERVER/deposit
+```
+
+Request Parameters:
 
 Name | Type | Description
 -----|------|------------
 `asset_code` | string | The code of the asset the user is wanting to deposit with the anchor. Ex BTC,ETH,USD,INR,etc
-`account` | string | The stellar account ID of the user that wants to deposit. This is where the asset token will be sent.
-`memo_type` | string | (optional) type of memo to attach to transaction, one of `text`, `id` or `hash`
+`account` | `G...` string | The stellar account ID of the user that wants to deposit. This is where the asset token will be sent.
+`memo_type` | string | (optional) type of memo that anchor should attach to the Stellar payment transaction, one of `text`, `id` or `hash`
 `memo` | string | (optional) value of memo to attach to transaction, for `hash` this should be base64-encoded.
 
-If the given `account` doesn't exist yet then the anchor will fund and thus create the account with at least enough lumens for the minimum reserve and the trust line. It is suggested that the anchor take some of the asset that is sent in to pay for these lumens. The anchor doesn't have the account's secret key so the trust line must still be created by the client before the anchor can send the remaining asset tokens to the give account. The anchor should listen to see when the client has established this trust line. Once the trust line is there the anchor should send the asset tokens to the account in Stellar.
+Example:
 
-If the anchor won't create new accounts for users then it should return an error if the given account doesn't exist yet.
+```
+GET https://api.example.com/deposit?asset_code=ETH&account=GACW7NONV43MZIFHCOKCQJAKSJSISSICFVUJ2C6EZIW5773OU3HD64VI
+```
 
-On success the endpoint should return `200 OK` HTTP status code and a JSON object with the following fields:
+If the given Stellar `account` doesn't exist yet then the anchor will fund and thus create the account with at least enough `XLM` for the minimum reserve and a trust line. It is suggested that the anchor take some of the asset that is sent in to pay for the `XLM`. The anchor should not list this as a fee, as the user is still receiving the value of the `XLM`. The anchor doesn't have the user account's secret key so the user must create a trust line to the anchor's asset before the anchor can send the remaining asset tokens to the user's account. The anchor should listen to see when the user has established this trust line. Once the trust line is there the anchor should send the asset tokens to the account in Stellar to complete the deposit.
```

Comment:
This paragraph is pretty dense.  There's a lot to process.  Maybe split it up into at least 2 parts?  Something like:

If the given Stellar `account` has balance of 0, the anchor will fund and thus technically create the account with at least enough `XLM` for the minimum reserve and a trust line. It is suggested that the anchor take some of the asset that is sent in to pay for the `XLM`.  The anchor should not list this minimal funding as a fee because the user still receives the value of the `XLM` in their account. 

Since the anchor doesn't have the user account's secret key, the *user* must create a trust line to the anchor's asset before the anchor can actually send the remaining asset tokens to the user's account. The anchor should listen to see when the user has established this trust line. Once the trust line is there, the anchor should send the asset tokens to the account in Stellar to complete the deposit.